neural_compressor/config.py

```diff
@@ -28,24 +28,31 @@
     "auto": "post_training_auto_quant",
     "dynamic": "post_training_dynamic_quant",
     "static": "post_training_static_quant",
-    "qat": "quant_aware_training",
+    "weight_only": "post_training_weight_only",
 }
 
 
 ops_schema = Schema({
     Optional('weight', default=None): {
         Optional('granularity'): And(
             list,
-            lambda s: all(i in ['per_channel', 'per_tensor'] for i in s)),
+            lambda s: all(i in ['per_channel', 'per_tensor', 'per_block'] for i in s)),
```

Comment:
shall we align the term with paper recommended, per_group?

Reply:
OK, I agree.